docs/topics/opt-in-requirements.md

```diff
@@ -297,16 +293,16 @@ and provide the explanation in the deprecation message.
 annotation class ExperimentalDateTime
 ```
 
-## Experimental status of the opt-in requirements
+## Pre-stable status of the opt-in requirements
```

Comment:
Maybe "Beta"? "Pre-stable" is vague; and since Beta is the last pre-stable level, this heading isn't likely to change anymore.

Reply:
Fixed.